app/Controllers/Talk.php

```diff
@@ -154,6 +154,29 @@ public function submit(): ResponseInterface
         return $this->response->setJSON(['success' => 'TALK_SUBMITTED'])->setStatusCode(201);
     }
 
+    public function getPendingTalksForSpeaker(): ResponseInterface
+    {
+        $talkModel = model(TalkModel::class);
+        $pendingTalks = $talkModel->getPendingForSpeaker($this->getLoggedInUserId());
+
+        $pendingTalks = $this->addAdditionalDataToTalks($pendingTalks);
+        return $this->response->setJSON(['pending_talks' => $pendingTalks])->setStatusCode(200);
```

Comment:
please remove the `pending_talks` tag so that the array is the top element.

And for some reason the `suggested_time_slot` has CamelCase Keys.
please change that into `snake_case`

```
            "suggested_time_slot": {
                "id": 2,
                "eventId": 1,
                "startTime": "2024-06-22 14:00:00",
                "duration": 45,
                "isSpecial": false
            }
```

Reply:
Yes, will do 👍